src/sentry/api/urls.py

```diff
@@ -695,18 +695,25 @@
                     MonitorStatsEndpoint.as_view(),
                     name="sentry-api-0-monitor-stats",
                 ),
+            ]
+        ),
+    ),
+    url(
+        r"^organizations/(?P<organization_slug>[^\/]+)/monitors/",
```

Comment:
placement of these seems fine for now since its close to the original monitors urls, but could you add a note here that when we delete the old monitors routes that we should probably include these routes into the larger `/organizations/` route tree below?